### PR TITLE
Contact form -- extra bits and bobs

### PR DESCRIPTION
**Pull request by mateahoward**
Changes have been made so that we keep our CSS file consistent, reduce the amount of code added in and improve responsiveness

### CODE REVIEW

File: contact.html
Comment (mateahoward) [inline contact.html:86]: I think it'd be nice if we included pronouns and different names to our forms so I've added this

File: assets/css/forms.css
Comment (mateahoward) [inline assets/css/forms.css:139]: - we replaced 'grid' with 'flex',

- we added a min-height so that our footer doesn't come up too far on the page, 

- we added max-width so that even on ultra-wide screens we keep the shape we wanted 
(this makes sure that our content width never goes above 1200px, this is useful for ultra-wide screens and makes sure our elements don't look displaced or a bit silly being too far apart)

- we added a margin: 0 auto; so that our content is always centered 
(this is the part that gives us the whitespace I mentioned above if the screen is ultra-wide)

File: assets/css/forms.css
Comment (mateahoward) [inline assets/css/forms.css:240]: - we removed multiple media queries and replaced it with just one
- we used percentage values to make sure our content automatically gets sized
- we reduced font sizing for titles and paragraphs

File: assets/css/forms.css
Comment (mateahoward) [inline assets/css/forms.css:214]: - we removed multiple media queries and replaced it with just one
- we used percentage values to make sure our content automatically gets sized
- we reduced font sizing for titles and paragraphs

**Comment by mateahoward** [comment]
Hey @SulthanNK, I've done some small changes to your form and tried my best to explain which ones I've added and why 😄 Try having a read trough and pls let me know if you understood what I meant 😅

I've also added you in as a reviewer, you'll have to pull down this code/branch and check it to see if it's all still good, after that you can press approve or just let me know if something got messed up 💪 

After it's been approved (if all is okay) we'll merge it in ⭐

File: assets/css/forms.css
Comment (SulthanNK) [reply assets/css/forms.css:214]: Wow, This is what I needed. Without this I ended up with multiple viewport and write each one individually.

File: contact.html
Comment (SulthanNK) [reply contact.html:86]: ohh, Got it!

File: assets/css/forms.css
Comment (SulthanNK) [reply assets/css/forms.css:139]: oh okay. the content in the page is less. so that why I thought no need to scrolling and did that. but, forgot about ultra-wide monitors 😅😬

**Review by SulthanNK — COMMENTED** [review]
Hey @mateahoward, Thank for those short explanations and it's really helpful for me that whats going on there. After that I pulled down this branch into local and checked it, no issues in froms and design. Looks cool ( LGTM - Looks Good To Me) 😅👌

**Review by SulthanNK — APPROVED** [review]
LGTM!

**Review by likiipedia — APPROVED** [review]

**Pull request merged**
